### PR TITLE
Fut 18 function makro

### PR DESCRIPTION
**Pull request by karollayna**

### CODE REVIEW

File: Future/DESCRIPTION [hunk 1]
```diff
@@ -9,6 +9,7 @@ License: GPL-2
 Encoding: UTF-8
```
Comment (darsoo) [inline Future/DESCRIPTION:9]: bump version

File: Future/R/macro.R [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+#' @param product string, name of used product
+#' @param weight numeric, weight of used product
+#'
+#' @examples
+#' macronutrients("Kawior", 50)
+#'
+#' @export
+macronutrients <- function(product, weight){
+
+  assertthat::assert_that(assertthat::is.string(product),
```
Comment (darsoo) [inline Future/R/macro.R:14]: Please use one assert method - in separate task fix all aserts FUT-19

**Review by darsoo — APPROVED** [review]

File: Future/DESCRIPTION (hunk 1 above)
Comment (karollayna) [reply Future/DESCRIPTION:9]: done

File: Future/R/macro.R (hunk 2 above)
Comment (karollayna) [reply Future/R/macro.R:14]: ok

**Pull request merged**